staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -183,6 +188,12 @@ type configController struct {
 	// in the computation of the nominalCL values.
 	// This is tracked because it is an input to the allocation adjustment algorithm.
 	nominalCLSum int
+
+	// MaxSeatsTracker tracks the maximum seats that should be allocatable from the
+	// work estimator for a given priority level. This controller does not enforce
+	// any limits on max seats stored in this tracker, it is up to the work estimator
+	// to set lower/upper limits on max seats (currently min=1, max=10).
+	MaxSeatsTracker
```

Comment:
If I understand correctly, it is not necessary to hold the configController's lock while accessing this field.  If I have this right then please move this field above the comment saying "This must be locked while accessing the later fields".

Reply:
done